src/js/flow/acquia_lift.variations.js

```diff
@@ -53,18 +53,15 @@
   };
```

Comment:
I am thinking renaming it to ```$_SESSION['acquia_lift_edit_mode']``` to be more descriptive.

How do I produce this trigger? I couldn't find any "start edit mode" that's sending me over to another page ...